fireant/tests/mock_dataframes.py

```diff
@@ -60,10 +60,10 @@ def rollup(dataframe, levels):
 
 # Mock DF with single metric column and no dimension
 no_dims_single_metric_df = pd.DataFrame(
-    np.array([
-        np.arange(1),
-    ]),
-    columns=['one'],
+        np.array([
```

Comment:
are all these added tabs necessary?  We have been using 4 spaces for indentation every where else, let's keep it at that please.

Reply:
This is PyCharm's default formatter.

Reply:
Should be 4 spaces, pep8 https://www.python.org/dev/peps/pep-0008/#indentation

Reply:
Not sure what caused it but I just pushed a fix.